src/style.css

```diff
@@ -1 +1,96 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+  local("Avenir"),
+  url(fonts/Avenir.ttc),
+  url(fonts/Avenir-Book.ttf),
+  url(fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: flex;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  box-sizing: border-box;
+  align-items: center;
+  transition: box-shadow 0.3s ease;
+}
+
+.key {
+  outline: none;
+  border: none;
+  padding: 0;
+  font-weight: 300;
+  color: #3D4E61;
+  font-family: Avenir, sans-serif;
```

Comment:
Move to html selector and there only inherit this property. Fix all cases if exist